ts/store/reducers/wallet/wallets.ts

```diff
@@ -215,11 +215,18 @@ export const bPayListSelector = createSelector(
 
 /**
  * return true if the payment method is visible in the wallet (the onboardingChannel
- * is IO or WISP)
+ * is IO or WISP) or if the onboardingChannel is undefined.
+ * We choose to show cards with onboardingChannel undefined to ensure backward compatibility
+ * with cards inserted before the field was added.
+ * Explicitly handling the undefined is a conservative choice, as the field should be an enum (IO, WISP, EXT)
+ * but it is a string and therefore we cannot be sure that incorrect data will not arrive.
+ *
  * @param pm
  */
 export const isVisibleInWallet = (pm: PaymentMethod) =>
-  pm.onboardingChannel === "IO" || pm.onboardingChannel === "WISP";
+  pm.onboardingChannel === "IO" ||
+  pm.onboardingChannel === "WISP" ||
+  pm.onboardingChannel === undefined;
```

Comment:
As my personal flavor, when a variable must be compared with a set of value I prefer to include them in an array (or Set) and then check if it includes that variable

```typescript
const visibleOnboardingChannels = ["IO", "WISP", undefined];
export const isVisibleInWallet = (pm: PaymentMethod): boolean =>
  visibleOnboardingChannels.some(
    oc => oc === pm.onboardingChannel?.toLowerCase().trim()
  );
```

what do you think? this code also is more strict for strings, forcing upper case

Reply:
I like it. Thank you!
Refactored in [e005005](https://github.com/pagopa/io-app/pull/3060/commits/e00500565757d27121e1c9e36ff1c50da313d491)